extensions/experience-platform-connector/src/events-collector/handleEvent.js

```diff
@@ -0,0 +1,24 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2022 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+import handlers from './handlers';
```

Comment:
It should be possible to install `@magento/experience-platform-connector` as a dependency and import the handlers from there so we don't have to duplicate a lot of code/tests.

Reply:
tried that already. the library does not allow deep imports.

Reply:
The package contains all the source files so you can try to do 

`import handleEvent from '@magento/experience-platform-connector/src/handleEvent'`

Reply:
yeah, got it working. it was a webpack config issue. @herzog31 pointed that out to me